src/index.js

```diff
@@ -69,6 +70,30 @@ function hydrator (inventory, installing, params, callback) {
     if (viewsDir && file.includes(viewsDir)) return false
     return true
   })
+
+  // Add deno cacheable files if they exist and we're hyrdrating a Deno runtime
```

Comment:
I'm not sure this change would be the right approach. How about adding these to the glob pattern, and then filter for Deno by entry files on Deno Lambdas?

Reply:
Yep, cool. No probs. Will push some amends later

Reply:
Does the latest commit work better?